src/app_unit.rs

```diff
@@ -185,6 +185,12 @@ impl<'a> Sum<&'a Self> for Au {
     }
 }
 
+impl<'a> Sum<Self> for Au {
```

Comment:
This should have produced a warning saying that the `<'a>` is unnecessary.

Reply:
Thanks. Addressed in #56.